insights/specs/default.py

```diff
@@ -214,7 +215,8 @@ def corosync_cmapctl_cmd_list(broker):
     @datasource(HostContext)
     def du_dirs_list(broker):
         """ Provide a list of directorys for the ``du_dirs`` spec to scan """
-        return ['/var/lib/candlepin/activemq-artemis']
+        filters = get_filters(Specs.du_dirs)
```

Comment:
@huali027 - would you please move this `datasource` to the `insights/specs/datasource` dir and add a test for it? Thanks.

Reply:
And adding the `filterable=True` is a kind of `API change` which will break the existing rules depending on it. Generally it requires an approve of "SPEC Change" in the assets repo.

Reply:
@xiangce, sure, I'll do it.